src/components/date-picker/date-picker.scss

```diff
@@ -1,233 +1,13 @@
-@import '../../style/internal/mdc-variables';
-
-@import '@limetech/mdc-theme/mdc-theme';
-@import '../../../node_modules/flatpickr/dist/flatpickr.min.css';
-
-$inactiveTextColor: var(
-    --mdc-theme-text-disabled-on-background,
-    #{$mdc-theme-text-disabled-on-background}
-);
-
-$datepickerHeaderColor: var(--lime-primary-color, #{$mdc-theme-primary});
-$datepickerSelectedColor: $datepickerHeaderColor;
-$datepickerHoverHighlightColor: $datepickerHeaderColor;
-$datepickerTextOnThemeColor: var(
-    --lime-on-primary-color,
-    #{$mdc-theme-on-primary}
-);
-
-.container {
-    display: inline-block;
-    position: relative;
-    width: 100%;
+limel-portal {
+    height: 3.5rem;
```

Comment:
The portal seems to adjust its height to the content, so I'm not quite sure why this height is set. Is it so that flatpickr will render correctly the first time, and the portal then adjusts after that?

Reply:
It's because I wanted it to render above the input element and not "over it" when rendering upwards

Reply:
Ah, of course! We shouldn't need the `height` value since we've got the `top:0` and `bottom:0` though, so I'd propose removing `height`.

Reply:
:zap